File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucThanhVien.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucThanhVien } from 'src/app/shared/DanhMucThanhVien.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucThanhVienService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucThanhVien";
    }
}

File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucThanhVienService.cs
namespace Service.Implement
{
    public class DanhMucThanhVienService : BaseService<DanhMucThanhVien, IDanhMucThanhVienRepository>
    , IDanhMucThanhVienService
    {
    private readonly IDanhMucThanhVienRepository _DanhMucThanhVienRepository;
    public DanhMucThanhVienService(IDanhMucThanhVienRepository DanhMucThanhVienRepository) : base(DanhMucThanhVienRepository)
    {
    _DanhMucThanhVienRepository = DanhMucThanhVienRepository;
    }
    }
    }

